docs/miscellaneous/contribute/translate/rewards.md

```diff
@@ -5,24 +5,58 @@ sidebar_position: 6
+## How our reward system works
 
-To check your translation progress, open a project you have worked on, click the *Reports* tab and select *My Contribution*. You can sum up the number of words translated in different projects to get a license.
+**For AdGuard licenses:**
 
-## Here’s how our reward system works
+- For your first one-year Ad Blocker license, you need to translate **1,000 words**.
+- For your second license, you’ll need an **additional 1,500 words** (a total of 2,500 words).
+- All subsequent licenses require an **additional 2,500** words per license.
 
-```text
-For the first one-year AdGuard license, you need to translate 500 words.
-For the second license, you need 1,000 words, i.e. 1,500 words altogether.
-All subsequent licenses require an additional 1,500 words per license.
+**For AdGuard VPN licenses:**
 
-For the first one-year AdGuard VPN subscription, you need to translate 2,500 words.
-All subsequent subscriptions or renewals require an additional 2,500 words per subscription.
+- The first one-year VPN subscription requires translating **3,000 words**.
+- All subsequent subscriptions or renewals require an **additional 3,000 words** per subscription.
 
-```
+:::note Limitations
+As a reward for contributing, We offer licenses and subscriptions valid for a maximum of 5 years. In the case of Ad Blocker, we offer licenses that cover up to 5 devices. We do not offer lifetime licenses.
 
-Just send us a request to [translate@adguard.com](mailto:translate@adguard.com) and you’ll get the license as a token of our gratitude.
+:::
 
-Besides, the most active translators may receive proofreader status in the future, should they express such an intention, and that promises even more good stuff.
+## How to request your license
 
-Every year we present gifts with the AdGuard logo to our active proofreaders. If you want to become one of them, please write to [translate@adguard.com](mailto:translate@adguard.com).
+Just send us an email to our translators team at [translate@adguard.com](mailto:translate@adguard.com) with the following information:
+
+- Your Crowdin username
+- The projects you contributed to (Websites, Applications, VPN Knowledge Base, etc.)
+- The email address to which you want to link your license
+- Your translation statistics (a screenshot or total word count) and the type of license you want
+
+**Email template:**
+
+> Hi AdGuard team!
+>
+> I’ve translated more than 5,000 words in AdGuard for Android in (*your language*) — you can find my stats attached.
+I would like to request an AdGuard VPN license and a one Ad Blocker license for that.
+My email is (*your email address*).
+My Crowdin username is (*your username*).
+
+## How to check your translation progress
+
+Open a project you have worked on, click the *Reports* tab, and select *My Contribution* Then scroll down, select a date range, and generate a report. It will look something like this.
```

Comment:
```suggestion
Open a project you have been working on, click the *Reports* tab, select *My Contribution*, scroll down, select a date range, and generate a report. It will look something like this.
```